clouddriver-aws/src/main/groovy/com/netflix/spinnaker/clouddriver/aws/deploy/ops/loadbalancer/UpsertAmazonLoadBalancerAtomicOperation.groovy

```diff
@@ -154,4 +158,30 @@ class UpsertAmazonLoadBalancerAtomicOperation implements AtomicOperation<UpsertA
     task.updateStatus BASE_PHASE, "Done deploying load balancers."
     operationResult
   }
+
+  private void updateApplicationSecurityGroupIngressRules(Map<String, String> securityGroupNamesToIds,
+                                                          ArrayList<Listener> listeners,
+                                                          RegionScopedProviderFactory.RegionScopedProvider regionScopedProvider) {
+    Names names = Names.parseName(description.getName())
+    String application = names.getApp()
+    try {
+      if (application in securityGroupNamesToIds) {
```

Comment:
Now I am thinking we could also look for the application security group if not provided in the request